content/en/security/access_control.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Access Control
+disable_toc: false
+products:
+- name: Cloud SIEM
+  url: /security/cloud_siem/
+  icon: siem
+- name: CSM Threats
+  url: /security/threats/
+  icon: cloud-security-management
+- name: Application Security Management
+  url: /security/application_security/
+  icon: app-sec
+further_reading:
+- link: "logs/processing/pipelines"
+  tag: "Documentation"
+  text: "Log processing pipelines"
+---
+
+{{< product-availability >}}
+
+## Overview
+
+Datadog's access management system uses role-based access control to let you determine the level of access users have to Datadog resources. Users are added to roles, which define the account permissions those users have, such as what data they can read or what account assets they can modify. When permissions are granted to a role, any user who is associated with that role receives those permissions. See the [Account Management Access Control][1] documentation for more information.
+
+For Datadog Security products, granular access control is available for detection rules and suppressions, so you can restrict access to them by teams, roles, or service accounts.
+
+## Permissions
+
+See the [list of permissions][2] for Security products.
+
+## Granular access control
+
+[Granular access control][3] is available for view and write permissions for detection and suppression rules.
```

Comment:
My suggestion would be to remove this section (it's repeating information that's mentioned in the Overview), and bump the heading levels for the following two sections to h2.